parser/src/parser.rs

```diff
@@ -253,6 +253,40 @@ mod tests {
         )
     }
 
+    #[test]
+    fn test_parse_tuples() {
+        let source = String::from("a, b = 4, 5\n");
+
+        assert_eq!(
+            parse_statement(&source),
+            Ok(ast::LocatedStatement {
+                location: ast::Location::new(1, 1),
+                node: ast::Statement::Assign {
+                    targets: vec![ast::Expression::Tuple {
```

Comment:
What kind of expressions are allowed at the left hand side (LHS)? Is it tuples, names only? Or are lists also allowed?

Reply:
Tuples, lists, names, attributes, indices, slices.

I'm not sure if there are any others.